site/documentation.markdown

```diff
@@ -22,6 +22,7 @@ If you are new to Haskell and are not sure where to start from, we recommend [CI
 *   \[$$\] [Haskell: The Craft of Functional Programming](http://www.haskellcraft.com/craft3e/Home.html)
 *   \[$$\] [The Haskell School of Music](http://euterpea.com/haskell-school-of-music/)
 *   \[$$\] [Get Programming with Haskell](https://www.manning.com/books/get-programming-with-haskell)
+*   [Learn Haskell by building a blog generator](https://lhbg-book.link)
```

Comment:
Looks like a great addition to me :).

Is the capitalization of the title intentional? I'd normally expect title case for a book (ie "Learn Haskell by Building a Blog Generator"), but it's perfectly fine to do something different if it's a stylistic choice or something.

Reply:
It is I guess, I don't know why I'm not a huge fan of capitalizing all words in a title 😅, but I don't mind changing that to fit the rest of the list.
FWIW the capitalization is similar to how 'Haskell Programming from first principles' is capitalized in the list.

Reply:
Since that's what you've chosen for the book itself, makes sense to keep it the same on the resources page too. I just wanted to double-check that this was what you intended :).